tool/tsh/tsh.go

```diff
@@ -146,6 +147,11 @@ func main() {
 	Run(cmd_line, false)
 }
 
+const (
+	clusterEnvVar = "TELEPORT_SITE"
```

Comment:
is this for internal use? the docs always refer to "sites" as "cluster", i.e. `TELEPORT_CLUSTER`?